site/src/server.rs

```diff
@@ -440,8 +440,18 @@ async fn serve_req(server: Server, req: Request) -> Result<Response, ServerError
         }
         "/perf/processed-self-profile" => {
             let ctxt: Arc<SiteCtxt> = server.ctxt.read().as_ref().unwrap().clone();
+            let support_brotli = req
```

Comment:
There is already `allow_compression` on line 342, let's reuse it. This one has a bug; it should check `ACCEPT_ENCODING`, not `CONTENT_ENCODING` :) Splitting it by commas is nicer than just the `contains` check in `allow_compression`, but I think that in that case we should check if the trimmed string starts with `br`, because there can be some [compression options](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#syntax).